libraries/classes/Plugins/Auth/AuthenticationKeycloak.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpMyAdmin\Plugins\Auth;
+
+use Lcobucci\JWT\Parser;
+use Lcobucci\JWT\Token;
+use PhpMyAdmin\Plugins\AuthenticationPlugin;
+
+/**
+ * Class AuthenticationKeycloak
+ * @package PhpMyAdmin\Plugins\Auth
+ */
+class AuthenticationKeycloak extends AuthenticationPlugin
+{
+    public function showLoginForm(): bool
+    {
+        return false;
+    }
+
+    public function readCredentials(): bool
+    {
+        if (!isset($_COOKIE["kc-access"])) {
```

Comment:
```suggestion
        if (!isset($_COOKIE['kc-access'])) {
```
`composer run phpcbf` will help you